PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -156,6 +165,30 @@ function transformReports(inputReportFiles: string[], index: number)
     }
 }
 
+function mapToAnalyzer(node: any) {
+    let ruleId = node.attributes.rule;
+    let violationType = node.name;
+    if (!ruleAnalyzerPairs.get(ruleId)) {
+        switch (violationType) {
+            case 'DupViol':
+                ruleAnalyzerPairs.set(ruleId, "com.parasoft.xtest.cpp.analyzer.static.dupcode");
+                break;
+            case 'FlowViol':
+                ruleAnalyzerPairs.set(ruleId, "com.parasoft.xtest.cpp.analyzer.static.flow");
+                break;
+            case 'MetViol':
+                ruleAnalyzerPairs.set(ruleId, "com.parasoft.xtest.cpp.analyzer.static.metrics");
+                break;
+            case 'StdViol':
```

Comment:
Let's use "default" here instead of "StdViol" to be consistent with TeamCity's code logic, which is also consistent with Jenkins.